templates/webpay/plus_mall_deferred/refund-form.html

```diff
@@ -0,0 +1,40 @@
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplo Webpay - Transaccion Refund</title>
+    {% include 'include/header.html' %}
+</head>
+
+<body class="container">
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="#">Ejemplo Webpay plus mall deferred- Transaccion Refund</a></li>
+        </ol>
+    </nav>
+
+    <div class="card">
+        <div class="card-body col-sm-4">
```

Comment:
Is the user supposed to input all these values?

Reply:
User can refund transaction input these values or in the confirmation view pushing "refund" button